### ISSUE TITLE
:bug: Footer is not properly loaded on the docs

### ISSUE THREAD

**Issue by mauromorales** [kairos-io/kairos#1737]
The footer in the docs, should look like the one on the homepage. The main reason why it is not, is because the one in the homepage adds all the content using the index.js asset, which is not loaded automatically by docsy.

The docsy footer has everything needed, IMO the best would be to just use the docsy footer as intened on the docs and also on the homepage

**Comment by mudler** [comment]
Is this still relevant @mauromorales ?

**Comment by mauromorales** [comment]
@mudler good catch, this was tackled in https://github.com/kairos-io/kairos-docs/pull/122, closing
